__tests__/fixtures/spec.json

```diff
@@ -387,6 +387,7 @@
             "name": "from",
             "required": true,
             "schema": {
+              "pattern": "[^\\/#\\?]+?",
```

Comment:
What's up with this change?

Reply:
This was because of updates in "path-to-regexp": "^6.2.1",